examples/data_sharing.tex

```diff
@@ -0,0 +1,211 @@
+
+\section{Deletion or return of \glsfmttext{type} Personal Data}
+\label{deletion}
+
+\subsection{Subject to this section \ref{deletion} \glsfmttext{processor} shall promptly and in any event within 10 business days of the date of cessation of any Services involving the Processing of \glsfmttext{type} Personal Data (the ``\textbf{Cessation Date}''), delete and procure the deletion of all copies of those \glsfmttext{undertaking} Personal Data.}
+
+\Glsfmttext{processor} shall provide written certification to \glsfmttext{undertaking} that it has fully complied with this section \ref{deletion} within 10 business days of the Cessation Date.
+
+\section{Audit Rights}
+\label{audits}
+
+\subsection{Subject to this section \ref{audits}, \glsfmttext{processor} shall make available to \glsfmttext{undertaking} on request all information necessary to demonstrate compliance with this Agreement, and shall allow for and contribute to audits, including inspections, by \glsfmttext{undertaking} or an auditor mandated by \glsfmttext{undertaking} in relation to the Processing of the \glsfmttext{undertaking} Personal Data by the Contracted Processors.}
+\label{audits:availability}
+
+Information and audit rights of \glsfmttext{undertaking} only arise under \autoref{audits:availability} to the extent that the Agreement does not otherwise give them information and audit rights meeting the relevant requirements of Data Protection Law.
+
+\section{Data Transfer}
+
+\subsection{The Processor may not transfer or authorize the transfer of Data to countries outside the EU and/or the European Economic Area (EEA) without the prior written consent of \gls{undertaking}.
+If personal data processed under this Agreement is transferred from a country within the European Economic Area to a country outside the European Economic Area, the Parties shall ensure that the personal data are adequately protected.
+To achieve this, the Parties shall, unless agreed otherwise, rely on EU approved standard contractual clauses for the transfer of personal data.}
+
+\section{General Terms}
+
+\subsection{\textbf{Confidentiality}.  Each Party must keep this Agreement and information it receives about the other Party and its business in connection with this Agreement (``\textbf{Confidential Information}'') confidential and must not use or disclose that Confidential Information without the prior written consent of the other Party except to the extend that:}
+
+\subsubsection{disclosure is required by law;}
+
+\subsubsection{the relevant information is already in the public domain.}
+
+\textbf{Notices.}  All notices and communications given under the Agreement must be in writing and will be delivered personally, sent by post or sent by email to the address or email address set out in the headin of this agreement at such other address as notified from time to time by the Parties changing their address.
```

Comment:
add a TODO for a notices env.

Reply:
Also, I didn't see any emails printed here, and I assume it's important, or people would be legally obliged to correspond via snail-mail for security breaches, which I find personally horrifying.

I've added commands for emails already - @passamo9, perhaps you could confirm where emails should go. Somewhere after the titlepage perhaps?